app/content/models/event.py

```diff
@@ -66,6 +65,10 @@ def __str__(self):
     def website_url(self):
         return f"/arrangementer/{self.id}/"
 
+    @property
+    def evaluation_url(self):
+        return f"sporreskjema/{self.evaluation.id}/"
```

Comment:
Du må ha en `/` på starten

```suggestion
        return f"/sporreskjema/{self.evaluation.id}/"
```